spec/compiler/formatter/formatter_spec.cr

```diff
@@ -1610,4 +1610,8 @@ describe Crystal::Formatter do
       # second comment
     }
     CODE
+
+  # 7272
+  assert_format "1 # => =>1+2"
```

Comment:
Better use an example that's explicitly about invalid code and doesn't look like there might be some special behaviour with `=> =>` :
```suggestion
  assert_format "1 # => ?invalid  code"
```